src/sentry/static/sentry/app/views/settings/projectAlerts/create.tsx

```diff
@@ -43,13 +46,17 @@ class Create extends React.Component<Props, State> {
   };
 
   componentDidMount() {
-    const {organization, project, location} = this.props;
+    const {
+      organization,
+      location,
+      params: {projectId},
+    } = this.props;
 
     trackAnalyticsEvent({
       eventKey: 'new_alert_rule.viewed',
       eventName: 'New Alert Rule: Viewed',
       organization_id: organization.id,
-      project_id: project.id,
+      project_id: projectId,
```

Comment:
project.id was a number projectId is the slug, probably didn't need to switch. Currently errors for me

Reply:
ooh good catch changing it back now

Reply:
Also getting this error
`Error: Unable to coerce value to number: 'internal'`

Reply:
Sorry, this error is gone. Tried `git pull` after seeing your other commits